package.json

```diff
@@ -13,7 +13,7 @@
     "build:examples": "pnpm run -r --parallel --if-present build:tokens",
     "changeset": "changeset",
     "dev": "pnpm run -r --parallel --if-present --filter \"@cobalt-ui/*\" dev",
-    "lint": "eslint \"packages/**/*.{ts,js,cjs,mjs}\"",
```

Comment:
There’s a current OOM bug where linting too many files from one process blows up. This splits the task into per-package lint rules (which will be helpful in the future if packages need additional linting)